Realm/Realm/Configurations/RealmConfigurationBase.cs

```diff
@@ -198,6 +209,7 @@ internal Native.Configuration CreateNativeConfiguration()
             return new Native.Configuration
             {
                 Path = DatabasePath,
+                FallbackPipePath = FallbackPipePath ?? string.Empty,
```

Comment:
I think we should leave that as `null` rather than `string.Empty`.

Reply:
And then we take care of null values in the setter? Or how does it work with passing null values to native?

Reply:
In the setter, we can pass a null string, we just need to use the `IntPtrLength` extension method to make sure we don't cause a NRE when reading the string length. Then in native we should check for `nullptr` like:

```cpp
if (configuration.fallback_path) {
  config.fifo_files_fallback_path = Utf16StringAccessor(configuration.fallback_path, configuration.fallback_path_len);
}
```